### ISSUE TITLE
Build fails on aarch64 - multiple definition of `tdesc_aarch64'

### ISSUE THREAD

**Issue by ggardet** [crash-utility/crash#66]
Build fails with LTO on aarch64 due to multiple definition of `tdesc_aarch64`:
```
[  467s] /usr/lib64/gcc/aarch64-suse-linux/10/../../../../aarch64-suse-linux/bin/ld: aarch64-linux-nat.o (symbol from plugin): in function `tdesc_aarch64':
[  467s] (.text+0x0): multiple definition of `tdesc_aarch64'; aarch64-tdep.o (symbol from plugin):(.text+0x0): first defined here
```

**Comment by ggardet** [comment]
And without LTO, there is also a similar problem:
```
[  680s] /usr/lib64/gcc/aarch64-suse-linux/10/../../../../aarch64-suse-linux/bin/ld: aarch64-linux-nat.o:(.bss+0x18): multiple definition of `tdesc_aarch64'; aarch64-tdep.o:(.bss+0x8): first defined here
```

**Comment by ggardet** [comment]
It could be fixed by https://github.com/crash-utility/crash/commit/e770735200c02ac2414c394ea6ec5f7f033efe64

**Comment by ggardet** [comment]
> It could be fixed by [e770735](https://github.com/crash-utility/crash/commit/e770735200c02ac2414c394ea6ec5f7f033efe64)

I confirm this commit fixes my problem.
